src/UI/Core/Label.cpp

```diff
@@ -0,0 +1,41 @@
+#include "Label.h"
+
+#include "../../Constants.h"
+#include "../../FontManager.h"
+
+#include "NAS2D/Renderer/Renderer.h"
+#include "NAS2D/Resources/Font.h"
+
+#include "NAS2D/MathUtils.h"
+#include "NAS2D/Utility.h"
+
+#include <locale>
+
+static const int FIELD_PADDING = 4;
+static NAS2D::Font* TXT_FONT = nullptr;
```

Comment:
This looks like it would be more suitable as a member variable rather than a `static` global.

Reply:
This is matching the code from many of the other derived controls. I did it mostly as a way to avoid getting pointers to a Font object every time it's needed. In the other controls they're all static local variables and are there literally to be able to grab a pointer once and then use the pointer where needed. The pointer can be updated as needed.

Reply:
It also has the side effect of changing the font for all controls that share the pointer at the same. It's not possible to have two different labels with two different fonts shown at the same time.

I agree with your assessment about the change being out of scope though.